tests/variable/variable.cpp

```diff
@@ -386,6 +386,154 @@ TEST(Variable, getNullParentComponent)
     EXPECT_EQ(nullptr, v->parent());
 }
 
+TEST(Variable, hasDirectEquivalentVariable)
+{
+    libcellml::VariablePtr v1 = std::make_shared<libcellml::Variable>();
+    libcellml::VariablePtr v2 = std::make_shared<libcellml::Variable>();
+    libcellml::VariablePtr v3 = std::make_shared<libcellml::Variable>();
+    libcellml::VariablePtr v4 = std::make_shared<libcellml::Variable>();
+    libcellml::VariablePtr v5 = std::make_shared<libcellml::Variable>();
+
+    EXPECT_FALSE(v1->hasDirectEquivalentVariable(v1));
```

Comment:
This test is way to verbose.  It should just do the minimal required to check for the direct equivalence.  This doesn't require 5 variables.

Reply:
Agreed, 3 variables are good enough. Addressed in 941c794.